appeals/web/src/client/components/show-more/show-more.js

```diff
@@ -103,7 +103,26 @@ const bindEvents = (/** @type {ShowMoreComponentInstance} */ componentInstance)
 		});
 };
 
-const initialiseTextMode = (/** @type {ShowMoreComponentInstance} */ componentInstance) => {
+const awaitInnerText = async (
+	/** @type {ShowMoreComponentInstance} */ componentInstance,
+	depth = 0
+) => {
+	if (componentInstance.elements.root.innerText || depth > 100) {
```

Comment:
Just a suggestion, the depth limit and timeout duration could be moved into named variables